client/doc.go

```diff
@@ -0,0 +1,154 @@
+// Package lxd implements a client for the LXD API
+//
+// Warning
+//
+// This API isn't considered STABLE yet!
```

Comment:
This comment will show up right at the top of the godoc page listed from README.md